config/locales/hyrax.en.yml

```diff
@@ -727,6 +730,7 @@ en:
         keyword:                   "Keyword"
         lease_expiration_date:     'until'
         related_url:               "Related URL"
+        title:                     "Title"
```

Comment:
Why are you adding this to defaults rather than admin_set?  I would think it would only make sense to be in defaults if there is more than one type that is using it.